### ISSUE TITLE
tutorial 1 mamba tool install error

### ISSUE THREAD

**Issue by jeffsun905** [NIGMS/DNA-Methylation-Sequencing-Analysis-with-WGBS#14]
Hi,

When i run this mamba install command, I got some errors and the pipeline tools appear not installed (the following fastqc not found...). What could be the reason?

Thank you,

! mamba install -y -c conda-forge -c bioconda \
    fastqc=0.11.9 \
    multiqc=1.13 \
    bismark=0.23.1 \
    trim-galore=0.6.7 \
    bedtools=2.30.0 \
    samtools=1.15.1 \
    metilene=0.2.8

**ERROR conda.core.link:_execute(949): An error occurred while installing package 'conda-forge::kiwisolver-1.4.5-py310hd41b1e2_1'.
Rolling back transaction: done
[Errno 13] Permission denied: '/opt/conda/lib/python3.10/site-packages/kiwisolver/__pycache__/__init__.cpython-310.pyc'
()
[Errno 13] Permission denied: '/opt/conda/lib/python3.10/site-packages/kiwisolver/__pycache__/__init__.cpython-310.pyc'
()**

**Comment by JohnTurnerNIH** [comment]
Hey Jeffsun,

Are you attempting to install this on Vertex AI or to a local Mamba install?

**Comment by jeffsun905** [comment]
Vertex AI. It is likely due to outdated documentation vs updated GCP environment. The error happened when I tried to create a notebook following the documentation but the current GCP environment does not have extact matches. When I selected some of GCP default settings, then it worked without that error. I also could not make the tutorial 4 (gcp batch) work due to updated GCP. The documentation likely needs to be updated frequenctly.
